Header header
int32 FWR_pulse
int32 FWL_pulse
int32 RWR_pulse
int32 RWL_pulse